backend/app/assets/stylesheets/spree/backend/shared/_tables.scss

```diff
@@ -4,27 +4,9 @@ table {
   border-spacing: 0;
 
   th, td {
-    border-right: 1px solid $color-border;
-    border-bottom: 1px solid $color-border;
-    vertical-align: middle;
+    vertical-align: top;
```

Comment:
Separate PR makes sense. I think we're due for another solution in that area entirely.